lib/skate/detours/detours.ex

```diff
@@ -114,9 +130,16 @@ defmodule Skate.Detours.Detours do
   def categorize_detour(%{state: %{"value" => %{"Detour Drawing" => "Past"}}}, _user_id),
     do: :past
 
-  def categorize_detour(_detour_context, nil = _user_id), do: :draft
-  def categorize_detour(%{author_id: author_id}, user_id) when author_id == user_id, do: :draft
-  def categorize_detour(_, _), do: nil
+    @doc """
+  Takes a `Skate.Detours.Db.Detour` struct and a `Skate.Settings.Db.User` id
+  and returns a `t:detour_type/0` based on the state of the detour.
+
+  otherwise returns `nil` if it is a draft but does not belong to the provided
+  user
+  """
```

Comment:
This @doc comment seems incorrect?

Reply:
Oh this was a very weird merging situation. Didn't want to delete the categorize_detour functions either. That should be in another PR

```suggestion
  def categorize_detour(_detour_context, nil = _user_id), do: :draft
  def categorize_detour(%{author_id: author_id}, user_id) when author_id == user_id, do: :draft
  def categorize_detour(_, _), do: nil
  
```